src/NServiceBus.Transport.RabbitMQ/Receiving/MessageHandlerContextExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace NServiceBus
+{
+    using RabbitMQ.Client.Events;
+    using System;
+
+    /// <summary>
+    /// Extensions to <see cref="IMessageHandlerContext"/>.
+    /// </summary>
+    public static class MessageHandlerContextExtensions
```

Comment:
why a new type for one method? i would just put it on `RabbitMQTransportSettingsExtensions`